package.js

```diff
@@ -0,0 +1,20 @@
+Package.describe({
+  name: 'useful:collection-mocks',
+  version: '0.0.1',
+  summary: 'Adds a method to Mongo.Collection\'s prototype that mocks the results of a db operation',
+  git: 'https://github.com/usefulio/meteor-collection-mocks.git',
```

Comment:
This may or may not be important, but what would be best practice for quick edits like this, fixing typos etc.? Should I commit straight to master, create a separate branch just for them or incorporate them into the next feature?